Tests/ApolloCodegenTests/CodeGeneration/Templates/PersistedQueriesOperationManifestTemplateTests.swift

```diff
@@ -201,20 +201,20 @@ class PersistedQueriesOperationManifestTemplateTests: XCTestCase {
       )
     ].map(OperationManifestItem.init)
 
-    let expected = """
+    let expected = #"""
       {
         "format": "apollo-persisted-query-manifest",
         "version": 1,
         "operations": [
           {
-            "id": "efc7785ac9768b2be96e061911b97c9c898df41561dda36d9435e94994910f67",
+            "id": "9db65faebf9e503b403964a81c90edbeeb894d46029b1b42b16639dda96772bd",
             "body": "query Friends { friends { ...Name } }\\nfragment Name on Friend { name }",
```

Comment:
And we are certain that the `\\n` is the correct way to represent the newline character in the JSON? It seems odd to me, but I see that you and @glasser discussed this in detail.

Reply:
The simplest thing might be to just run the thing outside of a test suite on a project with operations with fragments and share the output file here...

Reply:
Agreed, that's definitely something that could be implemented in a lot of our testing that might make things easier